src/components/AccountDetails/index.tsx

```diff
@@ -265,7 +264,7 @@ export default function AccountDetails({
               <AccountGroupingRow>
                 {formatConnectorName()}
                 <div>
-                  {connector !== injected && connector !== walletlink && (
+                  {connector !== injected && connector !== coinbaseWallet && (
```

Comment:
use instanceOf

Reply:
might be good to have a `type` per connector and then export an Enum or something?